src/app/lab/components/units-dropdown.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+import { Chart } from '../../models/chart.model';
+import { Indicator } from '../../models/indicator.model';
+
+/*  Units Dropdown Component
+
+    -- Requires handling unit selection
+    Expected use:
+        <ccl-units-dropdown
+            [indicator]="your_indicator"
+            [unit]="your_unit"
+            (unitSelected)="onUnitSelected($event)">
+*/
+
+@Component({
+  selector: 'ccl-units-dropdown',
+  template: `<div dropdown class="dropdown dropdown-units">
+              <button dropdownToggle class="button dropdown-toggle" type="button"
+                id="unitsDropdown" data-toggle="dropdown" aria-haspopup="true"
+                aria-expanded="true">
+                <i class="icon-flash"></i>
+                {{ unit || indicator.default_units }}
```

Comment:
I dunno. I think anyone looking at climate data through lab or the API would know what units they wanted. And for the ones they don't know, (basically, only K or kg/m^2) they probably don't want that anyway. Lastly how would you write kg/m^2 in a human readable way that is actually _more clear_ lol